chest/core.py

```diff
@@ -38,14 +42,33 @@ def _do_nothing(*args, **kwargs):
     pass
 
 
+class full_policy(object):
+    """ Behaviours to use when a a chest is full.
+    """
+    @staticmethod
+    def raise_(chest):
+        """A policy that raises an ENOSPC error if the disk is full.
+        """
+        raise OSError(errno.ENOSPC, os.strerror(errno.ENOSPC), chest.path)
+
+    @staticmethod
+    def pop_lru(chest):
+        """A policy that removes the most recently used element from
+        the cache when the chest is full.
+        """
+        del chest[chest.heap.popitem()[0]]
```

Comment:
I am using this only as a namespace. I normally agree that free functions are better than functions but it is nice that I can import `full_policy` and then discover all of the policies we have provided.

How would you feel about making this class namespace a submodule?